views/index.ejs

```diff
@@ -97,6 +97,18 @@
           <input type="text" id="paymentGateway" name='paymentGateway' value="">
         </td>
       </tr>
+      <tr>
+        <td>Locale:</td>
+        <td>
+          <select id="locale" class="field-select" name='locale'>
```

Comment:
The locale has been configured in the config.json. it should not be in the index.ejs

Reply:
Okay